modules/hitlnext/src/backend/service.ts

```diff
@@ -0,0 +1,100 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { Config } from '../config'
+import { MODULE_NAME } from '../constants'
+import { ExitTypes, IHandoff } from '../types'
+import { StateType } from '.'
+import Repository from './repository'
+
+export const toBpDestination = (botId: string, handoff: Pick<IHandoff, 'userId' | 'userThreadId' | 'userChannel'>) => {
```

Comment:
I find the naming, to BP destination, to be quite confusing. What is a BP destination?

```suggestion
export const toEventDestination = (botId: string, handoff: Pick<IHandoff, 'userId' | 'userThreadId' | 'userChannel'>) => {
```

Reply:
Done